### PR TITLE
Eng 42 add settings

### PR DESCRIPTION
**Pull request by E-zh**

### CODE REVIEW

File: src/Http/Controllers/Admin/StaticPagesController.php [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 namespace Ourgarage\StaticPages\Http\Controllers\Admin;
 
-use Ourgarage\StaticPages\Http\Requests\StaticPageCreateRequest;
 use App\Http\Controllers\Controller;
 use Ourgarage\StaticPages\Models\StaticPage;
+use Ourgarage\StaticPages\Http\Requests\StaticPageCreateRequest;
```
Comment (garf) [inline src/Http/Controllers/Admin/StaticPagesController.php:7]: For what?

**Comment by garf** [comment]
1 small comment. And can be merged.

File: src/Http/Controllers/Admin/StaticPagesController.php (hunk 1 above)
Comment (E-zh) [inline src/Http/Controllers/Admin/StaticPagesController.php]: To verify the input. Is not it so?

Comment (E-zh) [inline src/Http/Controllers/Admin/StaticPagesController.php:7]: To verify the input. Is not it so?

Comment (garf) [inline src/Http/Controllers/Admin/StaticPagesController.php:7]: I see no usage in this file :)

**Pull request merged**